src/panels/config/areas/dialog-area-registry-detail.ts

```diff
@@ -80,14 +78,15 @@ class DialogAreaDetail extends LitElement {
           <div class="form">
             ${entry
               ? html`
-                  <div>
-                    ${this.hass.localize(
+                  <ha-textfield
```

Comment:
Why did we make this a disabled textfield? @matthiasdebaat

Reply:
It was just a text "Area ID: {name}" before. Now it's a textfield to make it look more like a form in general and since it cannot be edited after creation.

Reply:
But since it can never be edited (even not on creation) it is weird that it is in a textfield? If something is in a textfield I expect to be able to edit it.

If it is disabled I expect it to be disabled because of a state, but editable when in another state?

Reply:
Shall I remove it or do you have another proposition?

Reply:
I would personally keep it just text, and align it with the form fields

Reply:
I have changed it to a form settings row. I couldn't find a closer match to match in a form field.